README.md

```diff
@@ -1,3 +1,48 @@
+
+
+
+
 # Jernl
 
-It's journaling, with a calendar. Sort of based on the old Seinfeld yarn re: keeping the chain going, partly a solution to not wanting to pay for Notion. Signup is free: [jernl.space/register](https://jernl.space/register).
\ No newline at end of file
+## Installation
+
+To install all of the template files, run the following script from the root of your project's directory:
+
+```
+bash -c "$(curl -s https://raw.githubusercontent.com/CFPB/development/main/open-source-template.sh)"
```

Comment:
What does this do? It looks like it just adds a bunch of generic open source starter files (CHANGELOG.md, CONTRIBUTING.md, etc) to your local instance. Although this repo could probably do with some of those files, you probably don't need them locally.

I'm also a little hesitant to encourage people piping `curl` responses directly into `bash`, for security reasons. Like yeah they could go check the script at the link above, before `curl`ing it, but I tend to just copypaste stuff off the internet with wild abandon so I'd expect others would as well haha.

Actually now that I look at it, it looks like the **Introduction** section is addressed more accurately below, so you can probably just remove this section entirely.